tests/test_ideogram.py

```diff
@@ -629,7 +636,7 @@ def assert_callback(prop_value, nclicks, input_value):
 
     # assert the presence of homology region
     regions = wait_for_elements_by_css_selector(driver, '.syntenicRegion')
-    assert len(regions) != 0
+    assert len(regions) > 0
```

Comment:
Why was this changed? I guess it makes sense, since the length of a list cannot be negative, but I'm not sure this change affects performance or functionality. (Correct me if I'm wrong!)

Reply:
It shouldn't affect performance or functionality, but if ever a length was negative, I would want to catch it!! Well, I'm for writing the meaningful version directly (we expect this length to be positive, so let's write it, it's easier on the reader too).